src/jukebox/components/timers/__init__.py

```diff
@@ -77,6 +79,15 @@ def finalize():
     timer_fade_volume.__doc__ = "Timer step-wise volume fade out and shutdown"
     plugin.register(timer_fade_volume, name='timer_fade_volume', package=plugin.loaded_as(__name__))
 
+    global timer_idle_shutdown
+    timeout = cfg.setndefault('timers', 'idle_shutdown', 'timeout_sec', value=None)
```

Comment:
Can we use the definition of `timeout_sec=0` as a default to disable the timeout instead of commenting it out? I still believe that the non-existence of this config should still be handled (especially due to the fact that all jukebox.yaml's currently don't have this entry)

Reply:
Done.